src/test/java/apoc/util/FileUtilsRelativePathTest.java

```diff
@@ -0,0 +1,59 @@
+package apoc.util;
+
+import apoc.config.Config;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.test.TestGraphDatabaseFactory;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+
+public class FileUtilsRelativePathTest {
+    private GraphDatabaseService db;
+    private File testFile;
+
+    @Before
+    public void setUp() throws Exception {
+        db = new TestGraphDatabaseFactory().newImpermanentDatabaseBuilder()
+                .setConfig("apoc.import.file.use_neo4j_config", "true")
+                .setConfig("dbms.directories.import", "build/resources/test")
+                .setConfig("dbms.security.allow_csv_import_from_file_urls","true")
+                .setConfig("foo", "bar").setConfig("foouri", "baruri")
+                .setConfig("foopass", "foopass").setConfig("foo.credentials", "foo.credentials").newGraphDatabase();
+        TestUtil.registerProcedure(db, Config.class);
+        testFile = new File(Thread.currentThread().getContextClassLoader().getResource("test.csv").getPath());
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        db.shutdown();
+    }
+
+    @Test
+    public void changeNoSlashesUrlWithDirectoryImportContrainedURI() throws Exception {
+        assertEquals(new File(testFile.getAbsolutePath()).toURI().toString(), FileUtils.changeFileUrlIfImportDirectoryConstrained("test.csv"));
```

Comment:
Perché facciamo questo?
E non questo?
```suggestion
        assertEquals(testFile.toURI().toString(), FileUtils.changeFileUrlIfImportDirectoryConstrained("test.csv"));
```